libs/shared/ui/src/lib/components/button/button.tsx

```diff
@@ -16,8 +16,9 @@ const buttonVariants = cva(['inline-flex', 'items-center', 'shrink-0', 'font-med
     },
     size: {
       xs: ['text-xs', 'px-1', 'py-0.5'],
-      sm: ['text-sm', 'py-1', 'px-2'],
-      md: ['text-base', 'py-1.5', 'px-4'],
+      sm: ['text-sm', 'h-7', 'px-2'],
+      md: ['text-sm', 'h-9', 'px-3'],
+      lg: ['text-sm', 'h-10', 'px-4'],
```

Comment:
I added a fixed height to prevent the container from taking up the full height when using flexbox (where the parent sets the full height). Additionally, we need to avoid adding padding to the top and bottom because we don't have pixel-perfect sizing in this case

Reply:
Because, we use it for the Commit buttons, and it causes an issue, I can take look after